### DISCUSSION TITLE
Change icon color

### DISCUSSION THREAD

**Discussion by ghost** [hardcore-sushi/DroidFS#276] [Ideas]
Change app theme with icon color

Like theme  is red than icon color is red 

![Screenshot_20240307_191956_DroidFS.jpg](https://github.com/hardcore-sushi/DroidFS/assets/162563252/df1eb023-e009-4383-8355-0c529fca9585)

👆Theme is red but icon is green 👇

![Screenshot_20240307_192019_One UI Home.jpg](https://github.com/hardcore-sushi/DroidFS/assets/162563252/28d1210d-4851-4266-bb2f-221f7c9d738e)

🙏Please add this new feature

**Comment by hardcore-sushi** [comment]
Yes this would be a cool feature but not the most important in my opinion. It may be implemented one day.

**Reply by ghost** [reply]
Thank you 😊
